x/stakeibc/keeper/host_zone.go

```diff
@@ -35,6 +35,18 @@ func (k Keeper) GetHostZone(ctx sdk.Context, chainId string) (val types.HostZone
 	return val, true
 }
 
+// GetActiveHostZone returns an error if the host zone is not found or if it's found, but is halted
```

Comment:
Useful, we do this check everywhere so this should save many LOC across the codebase